### ISSUE TITLE
Latest sanic contains breaking change

### ISSUE THREAD

**Issue by gluap** [trustpilot/python-sanicargs#17]
The ``abort`` function imported from ``sanic`` here
https://github.com/trustpilot/python-sanicargs/blob/78f2cdccc959dcf5d301ceee669db8606b35b261/sanicargs/__init__.py#L9
was deprecated some time ago and removed with the LTS 21.12 version of sanic.

As a result an ``ImportError`` is raised when running against ``sanic>=21.12.0``

```
     from sanicargs import parse_parameters
   File "/usr/local/lib/python3.9/site-packages/sanicargs/__init__.py", line 9, in <module>
     from sanic.exceptions import abort
 ImportError: cannot import name 'abort' from 'sanic.exceptions' (/usr/local/lib/python3.9/site-packages/sanic/exceptions.py)
```
This could be circumvented by either creating a copy of the [missing abort function](https://github.com/sanic-org/sanic/blob/8673021ad4a159e2c12cd4886d5b26ba08a6fe38/sanic/exceptions.py#L245) or switching to the new normal of raising the specific exception with a message directly, e.g. ``InvalidUsage(message="Bad or missing value for %s" % name, status_code=400)``.

I tried doing this myself but trying to run the unit test suite it seems that the unit tests also do not survive an update of dependency packages, which is where I stopped and decided to file an issue instead - in the hopes that someone more versed with ``python-sanicargs`` will have an easier time disentangeling those.

The easy fix will of course be to just depending on sanic <21.12 for the time being.

**Comment by gluap** [comment]
Dug a bit more and the failing unit tests seem to - at least partly - fail on other packages that themselves break on the sanic update - https://github.com/yunstanford/pytest-sanic/issues/62 for instance seems related.

**Comment by sneeu** [comment]
Thanks for your report @gluap. I’m not sure if you resolved this another way but I’ve just published version 3.0 which should resolve your issues.
